docs/Voice/receiving.mdx

```diff
@@ -109,7 +110,6 @@ async def stop_recording(ctx):
     if ctx.guild.id in connections:  # Check if the guild is in the cache.
         vc = connections[ctx.guild.id]
         vc.stop_recording()  # Stop recording, and call the callback (once_done).
-        del connections[ctx.guild.id]  # Remove the guild from the cache.
```

Comment:
Why is this part removed?

Reply:
I re-added it on "once" function. The reason why i moved it bcz of "once" function is async. So when we call it and delete dict, dict is removed before using "await connections[channel.guild.id].disconnect()".

Reply:
Oh i noticed that, i forget something to add!